fs_image/static/src/views/fields/fsimage_field.xml

```diff
@@ -62,4 +62,10 @@
         </div>
     </t>
 
+    <t t-name="fs_image.ImageZoomTooltip">
```

Comment:
Weird error if try using `web.ImageZoomTooltip`. so, i have to added this new template

![Screenshot 2024-02-28 at 09-37-48 Odoo - New](https://github.com/nguyenminhchien/storage/assets/8724793/cf3148ee-6b5c-4a30-acfa-e008c0410c36)